fix(clerk-js): Expand Keyless UI if user is signed-in (#4969)
clerk/javascript: .changeset/strange-rings-applaud.md

---
'@clerk/clerk-js': minor
---

> Expand Keyless popover on initial render if user signs-in.

Reviewer: ```suggestion
Expand Keyless popover when a user signs-in.
```
